src/classes/Main.cls

```diff
@@ -0,0 +1,12 @@
+public with sharing class Main {
+	
+	 public static void main() {
+	 	
+	 	String x = '';
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.